woocommerce/wc-admin
# Replace deprecated isPrimary, isSecondary, isTertiary, isLink Button props with variant

Fixes #8313

This PR updates the deprecated isPrimary, isSecondary, isTertiary, isLink Button props.

> isPrimary, isSecondary, isTertiary and isLink props in Button have been deprecated. Use variant instead (https://github.com/WordPress/gutenberg/pull/31713).

Document Reference: https://github.com/WordPress/gutenberg/tree/trunk/packages/components/src/button#variant

### Screenshots

![Screen Shot 2022-02-22 at 14 57 27](https://user-images.githubusercontent.com/4344253/155079245-7478ce3c-c64d-454d-ba24-db9fcb336ad1.png)


primary, link

![Screen Shot 2022-02-22 at 14 40 39](https://user-images.githubusercontent.com/4344253/155078044-817aa8bb-3dc6-4396-b26e-06201bbfb85c.png)

primary, secondary, tertiary

### Detailed test instructions:

1. Checkout this branch
2. Go to the setup wizard
3. Oberse that the "Skip setup store details" button is shown as a link
2. Go to the Theme step
3. Observe that buttons are shown properly with the different variants

Review the other code changes
